app/assets/javascripts/misc/verify-modal.js

```diff
@@ -0,0 +1,42 @@
+const I18n = window.LoginGov.I18n;
```

Comment:
I've seen this `const` declaration a couple times now in separate .js files -- is there any concern about creating variables in the global scope and colliding namespaces?